### PR TITLE
DOCSP-33632 Filtered Subset

### PR DESCRIPTION
**Pull request by jordan-smith721**
# Pull Request Info

[PR Reviewing Guidelines](https://github.com/mongodb/docs-node/blob/master/REVIEWING.md)

JIRA - https://jira.mongodb.org/browse/DOCSP-33632
Staging - https://preview-mongodbjordansmith721.gatsbyjs.io/node/DOCSP-33632-filtered-top-subset/aggregation-tutorials/filtered-subset/

## Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [x] Did you run a spell-check?
- [x] Did you run a grammar-check?
- [x] Are all the links working?
- [x] Are the [facets and meta keywords](https://wiki.corp.mongodb.com/display/DE/Docs+Taxonomy) accurate?

### CODE REVIEW

File: source/aggregation-tutorials/filtered-subset.txt [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: tutorial
+
+.. meta::
+   :keywords: code example, node.js, sort, limit, aggregation
```
Comment (rustagir) [inline source/aggregation-tutorials/filtered-subset.txt:18]: Q: are the meta tags supposed to be directly under the heading? this is how I have seen it in other docs, but again not sure if it matters

File: source/aggregation-tutorials/filtered-subset.txt [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+
+Introduction
+------------
+
+In this tutorial, you can learn how to use the {+driver-short+} to
+find a subset of documents in a collection by using aggregation.
```
Comment (rustagir) [inline source/aggregation-tutorials/filtered-subset.txt:24]: S: also explain that this tutorial shows how to format the output using sorts, setting/unsetting fields, etc

Something like:
```suggestion
In this tutorial, you can learn how to use the {+driver-short+} to
find a subset of documents in a collection and format results by using aggregation.
```

File: source/aggregation-tutorials/filtered-subset.txt [hunk 3]
```diff
@@ -0,0 +1,194 @@
...
+
+.. tip::
+
+   You can also query for a subset of documents in a collection by using MongoDB
+   Query Language (MQL). To learn how to specify a query using MQL, see the 
+   :ref:`read operation guides <node-read-operations>`.
```
Comment (rustagir) [inline source/aggregation-tutorials/filtered-subset.txt:30]: i: MQL was renamed to the Query API -- can link some docs if needed. 

```suggestion
   You can also query for a subset of documents in a collection by using the Query API.
   To learn how to specify a query, see the :ref:`Read Operations guides <node-read-operations>`.
```

**Review by rustagir — CHANGES_REQUESTED** [review]
a few things, happy to chat about the open questions!

File: source/aggregation-tutorials.txt [hunk 4]
```diff
@@ -19,8 +19,9 @@ Aggregation Tutorials
 
 .. toctree::
 
-   /aggregation-tutorials/one-to-one-join/
+   /aggregation-tutorials/one-to-one-join.txt
    /aggregation-tutorials/multi-field-join/
+   /aggregation-tutorials/filtered-subset.txt
```
[the remark this answers is not in the document]
Comment (jordan-smith721) [reply source/aggregation-tutorials.txt:24]: I don't think it makes a difference so either way works. Changed to match the landing page.

File: source/aggregation-tutorials/filtered-subset.txt (hunk 1 above)
Comment (jordan-smith721) [reply source/aggregation-tutorials/filtered-subset.txt:18]: I'm not sure if there's a specific place other than at the top of the file, I just went with what I saw in other pages that we've added taxonomy tagging to.

File: source/aggregation-tutorials/filtered-subset.txt [hunk 5]
```diff
@@ -0,0 +1,194 @@
...
+After inserting the data into the collection, create a compound index on the
+``vocation`` and ``dateofbirth`` fields to
+improve query performance, as shown in the following code:
+
+.. literalinclude:: /includes/aggregation/filtered-subset.js
+   :language: javascript
+   :copyable: true
+   :start-after: start-index
+   :end-before: end-index
+   :dedent:
```
[the remark this answers is not in the document]
Comment (jordan-smith721) [reply source/aggregation-tutorials/filtered-subset.txt:82]: Messaged in Slack -- I deleted this for the reasons you mentioned and to simplify the tutorial.

**Review by rustagir — APPROVED** [review]
lgtm!

**Pull request merged**